packages/client/src/Call.ts

```diff
@@ -595,6 +596,30 @@ export class Call {
     return this.state.createdBy?.id === this.currentUserId;
   }
 
+  /**
+   * Update from the call response from the "call.ring" event
+   * @internal
+   */
+  updateFromRingingEvent = async (event: CallRingEvent) => {
+    await this.setup();
+    // call.ring event excludes the call creator in the members list
+    // as the creator does not get the ring event
+    // so update the member list accordingly
+    const creator = this.state.members.find(
+      (m) => m.user.id === event.call.created_by.id,
+    );
+    if (!creator) {
+      this.state.setMembers(event.members);
+    } else {
+      this.state.setMembers([creator, ...event.members]);
+    }
```

Comment:
Alternatively we could skip setting members altogether because call.created event already would have set members